explorer/explorer.go

```diff
@@ -793,41 +431,34 @@ func (exp *explorerUI) Store(blockData *blockdata.BlockData, _ *wire.MsgBlock) e
 	return nil
 }
 
+func (exp *explorerUI) StoreMPData(data *mempool.MempoolData, timestamp time.Time) error {
+	exp.MempoolData.RLock()
+	exp.MempoolData.NumTickets = data.NumTickets
+	exp.MempoolData.RUnlock()
+	exp.wsHub.HubRelay <- sigMempoolUpdate
+
+	return nil
+}
+
 func (exp *explorerUI) addRoutes() {
 	exp.Mux.Use(middleware.Logger)
 	exp.Mux.Use(middleware.Recoverer)
 	corsMW := cors.Default()
 	exp.Mux.Use(corsMW.Handler)
 
-	exp.Mux.Get("/", exp.root)
-	exp.Mux.Get("/ws", exp.rootWebsocket)
-
-	exp.Mux.Route("/block", func(r chi.Router) {
-		r.Route("/{blockhash}", func(rd chi.Router) {
-			rd.Use(exp.blockHashPathOrIndexCtx)
-			rd.Get("/", exp.blockPage)
-			rd.Get("/ws", exp.rootWebsocket)
-		})
-	})
-
-	exp.Mux.Route("/tx", func(r chi.Router) {
-		r.Route("/{txid}", func(rd chi.Router) {
-			rd.Use(transactionHashCtx)
-			rd.Get("/", exp.txPage)
-			rd.Get("/ws", exp.rootWebsocket)
-		})
-	})
-	exp.Mux.Route("/address", func(r chi.Router) {
-		r.Route("/{address}", func(rd chi.Router) {
-			rd.Use(addressPathCtx)
-			rd.Get("/", exp.addressPage)
-			rd.Get("/ws", exp.rootWebsocket)
-		})
-	})
-	exp.Mux.Route("/decodetx", func(r chi.Router) {
-		r.Get("/", exp.decodeTxPage)
-		r.Get("/ws", exp.rootWebsocket)
-	})
-
-	exp.Mux.With(searchPathCtx).Get("/search/{search}", exp.search)
+	redirect := func(url string) func(http.ResponseWriter, *http.Request) {
```

Comment:
a style thing since the underlying types are the same, but I'd like the return as `http.HandlerFunc` since that's what `chi.(*Mux).Get` takes, plus it's less repetition of the function signature here.